doc/configuration.rst

```diff
@@ -410,29 +410,29 @@ point to the directory containing ``searchindex.js``, such as
 If you wish to do the same for ordinary reST documentation,
 see :ref:`plain_rst`.
 
-If you are using inheritance for your documented code and you are experience
-wrong links in the sense that the links point to the base class of an object
+If you are using inherited classes in your code and are experiencing
+incorrect links in the sense that links point to the base class of an object
 instead of the child, the option ``prefer_full_module`` might solve your issue.
-Have also a look at `the GitHub
+See `the GitHub
 issue <https://github.com/sphinx-gallery/sphinx-gallery/issues/947>`__
-implementing this option for more context.
+for more context.
 
 To make this work in your documentation you need to include to the
-configuration
-dictionary within your Sphinx ``conf.py`` file::
+``prefer_full_module`` configuration within your Sphinx ``conf.py`` file::
 
     sphinx_gallery_conf = {
         ...
         'prefer_full_module':[
-        # a list of regex command of your module where the full module
-        # name should be used for sphinx_gallery instead of the shortend
-        'yourmodule.*+\d{4}',
```

Comment:
I was not sure exactly what this regex was intended to do, regex101 gave:

![image](https://github.com/sphinx-gallery/sphinx-gallery/assets/23182829/7369cc56-2468-4eb4-b26e-c9670019185c)

Have amended to just `mod.submod`  as simple example.